js/components/oppfolgingsdialog/godkjennplan/releasetplan/GodkjentPlan.js

```diff
@@ -37,6 +38,14 @@ const textBothApprovedOppfolgingsplan = (arbeidstakerName) => {
     return `Denne versjonen av planen er godkjent av ${arbeidstakerName} og deg.`;
 };
 
+const textForcedApprovedOppfolgingsplan = (rootUrl, arbeidstakerName) => {
```

Comment:
Om dette skal være en egen komponent, så er det greit å skrive det som en react-komponent med JSX i stedet for som en ren js-funksjon, da får man proptypes med på kjøpe som validere input.